src/watchdog.rs

```diff
@@ -1,7 +1,7 @@
 //! Traits for interactions with a processors watchdog timer.
 
 /// Feeds an existing watchdog to ensure the processor isn't reset. Sometimes
-/// commonly referred to as "kicking" or "refreshing".
+/// the "feeding" operation is commonly referred to as "refreshing".
```

Comment:
```suggestion
/// the "feeding" operation is referred to as "refreshing".
```

Reply:
Or "also".

Reply:
Thanks! 'Also' may be better here; pushed in a new commit.